assignments/assignment01.rb

```diff
@@ -37,14 +37,14 @@ def number_to_string(n, lang)
 
 # it accepts a string
 # and returns the same string with each word capitalized.
-def titleize(s)
-  # your implementation here
-end
 
-# Your method should generate the following results:
-titleize "hEllo WORLD"          #=> "Hello World"
+def titleize(input)
 
-titleize "gooDbye CRUel wORLD"  #=> "Goodbye Cruel World"
+  word = input.split.each { |x| x.downcase.capitalize! }.join(' ')
+  
+  puts word.chomp
```

Comment:
You want to return the value, not print it out. So intead of "puts word.chomp", you just want "word.chomp"